packages/clay-tabs/src/TabPane.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React, {useEffect, useState} from 'react';
+
+interface ITabPaneProps extends React.HTMLAttributes<HTMLDivElement> {
+	/**
+	 * Flag to indicate if `active` classname should be applied
+	 */
+	active?: boolean;
+
+	/**
+	 * Flag to indicate if `fade` classname that applies a fading animation should be applied.
+	 */
+	fade?: boolean;
+}
+
+const delay = (fn: Function) => {
+	return setTimeout(() => {
+		fn();
+	}, 100);
```

Comment:
Should we be hardcoding a specific delay for animations, or should we be using `onAnimationEnd`? Typically I remember trying to avoid hardcoded `setTimeout` for animations.

Reply:
I think this case is peculiar because in fact it is just a delay for as soon as the component render it adds the class and then it would cause an `onAnimationEnd`.